jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>If this were a textbook, this would be in a side box - people don't have to read it, and it's making the whole tutorial too long. But that's just personal opinon</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>